maps/PlayersHouse1F.asm

```diff
@@ -290,14 +290,14 @@ UnknownText_0x7a850:
 	line "convenient?"
 	done
 
-UnknownText_0x7a8b5:
+HurryUpElmIsWaitingText:
 	text "PROF.ELM is wait-"
 	line "ing for you."
 
 	para "Hurry up, baby!"
 	done
 
-UnknownText_0x7a8e5:
+AfterGotAMonText:
```

Comment:
"AfterGotAMonText" sounds like something that would be used by Elm in the cutscene after you pick a starter. Can it follow the same pattern as the rest, and directly reference what's being said, e.g. "SoWhatWasProfElmsErrandText"?